<!DOCTYPE html><html lang="en" ><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><meta name="generator" content="Jekyll v4.2.2" /><meta property="og:title" content="Wellcome Single Cell Conference 2022" /><meta property="og:locale" content="en" /><meta name="description" content="Ferran Cardoso`s personal profile, portfolio of academical publications, and other outputs. Uses a minimal, responsive, and powerful Jekyll theme for presenting professional writing." /><meta property="og:description" content="Ferran Cardoso`s personal profile, portfolio of academical publications, and other outputs. Uses a minimal, responsive, and powerful Jekyll theme for presenting professional writing." /><link rel="canonical" href="https://fercrcode.github.io/posts/WellcomeSC22/" /><meta property="og:url" content="https://fercrcode.github.io/posts/WellcomeSC22/" /><meta property="og:site_name" content="Ferran Cardoso" /><meta property="og:type" content="article" /><meta property="article:published_time" content="2022-06-05T01:00:00+01:00" /><meta name="twitter:card" content="summary" /><meta property="twitter:title" content="Wellcome Single Cell Conference 2022" /><meta name="twitter:site" content="@FerranC96" /> <script type="application/ld+json"> {"@context":"https://schema.org","@type":"BlogPosting","dateModified":"2023-06-09T22:20:35+01:00","datePublished":"2022-06-05T01:00:00+01:00","description":"Ferran Cardoso`s personal profile, portfolio of academical publications, and other outputs. Uses a minimal, responsive, and powerful Jekyll theme for presenting professional writing.","headline":"Wellcome Single Cell Conference 2022","mainEntityOfPage":{"@type":"WebPage","@id":"https://fercrcode.github.io/posts/WellcomeSC22/"},"url":"https://fercrcode.github.io/posts/WellcomeSC22/"}</script><title>Wellcome Single Cell Conference 2022 | Ferran Cardoso</title><link rel="apple-touch-icon" sizes="180x180" href="/assets/img/favicons/apple-touch-icon.png"><link rel="icon" type="image/png" sizes="32x32" href="/assets/img/favicons/favicon-32x32.png"><link rel="icon" type="image/png" sizes="16x16" href="/assets/img/favicons/favicon-16x16.png"><link rel="manifest" href="/assets/img/favicons/site.webmanifest"><link rel="shortcut icon" href="/assets/img/favicons/favicon.ico"><meta name="apple-mobile-web-app-title" content="Ferran Cardoso"><meta name="application-name" content="Ferran Cardoso"><meta name="msapplication-TileColor" content="#da532c"><meta name="msapplication-config" content="/assets/img/favicons/browserconfig.xml"><meta name="theme-color" content="#ffffff"><link rel="preconnect" href="https://fonts.googleapis.com" ><link rel="dns-prefetch" href="https://fonts.googleapis.com" ><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link rel="dns-prefetch" href="https://fonts.gstatic.com" crossorigin><link rel="preconnect" href="https://fonts.googleapis.com" ><link rel="dns-prefetch" href="https://fonts.googleapis.com" ><link rel="preconnect" href="https://cdn.jsdelivr.net" ><link rel="dns-prefetch" href="https://cdn.jsdelivr.net" ><link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Lato&family=Source+Sans+Pro:wght@400;600;700;900&display=swap"><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4/dist/css/bootstrap.min.css"><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.11.2/css/all.min.css"><link rel="stylesheet" href="/assets/css/style.css"><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/afeld/bootstrap-toc@1.0.1/dist/bootstrap-toc.min.css"><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/magnific-popup@1/dist/magnific-popup.min.css"> <script src="https://cdn.jsdelivr.net/npm/jquery@3/dist/jquery.min.js"></script> <script type="text/javascript"> class ModeToggle { static get MODE_KEY() { return "mode"; } static get MODE_ATTR() { return "data-mode"; } static get DARK_MODE() { return "dark"; } static get LIGHT_MODE() { return "light"; } static get ID() { return "mode-toggle"; } constructor() { if (this.hasMode) { if (this.isDarkMode) { if (!this.isSysDarkPrefer) { this.setDark(); } } else { if (this.isSysDarkPrefer) { this.setLight(); } } } let self = this; /* always follow the system prefers */ this.sysDarkPrefers.addEventListener("change", () => { if (self.hasMode) { if (self.isDarkMode) { if (!self.isSysDarkPrefer) { self.setDark(); } } else { if (self.isSysDarkPrefer) { self.setLight(); } } self.clearMode(); } self.notify(); }); } /* constructor() */ get sysDarkPrefers() { return window.matchMedia("(prefers-color-scheme: dark)"); } get isSysDarkPrefer() { return this.sysDarkPrefers.matches; } get isDarkMode() { return this.mode === ModeToggle.DARK_MODE; } get isLightMode() { return this.mode === ModeToggle.LIGHT_MODE; } get hasMode() { return this.mode != null; } get mode() { return sessionStorage.getItem(ModeToggle.MODE_KEY); } /* get the current mode on screen */ get modeStatus() { if (this.isDarkMode || (!this.hasMode && this.isSysDarkPrefer)) { return ModeToggle.DARK_MODE; } else { return ModeToggle.LIGHT_MODE; } } setDark() { $('html').attr(ModeToggle.MODE_ATTR, ModeToggle.DARK_MODE); sessionStorage.setItem(ModeToggle.MODE_KEY, ModeToggle.DARK_MODE); } setLight() { $('html').attr(ModeToggle.MODE_ATTR, ModeToggle.LIGHT_MODE); sessionStorage.setItem(ModeToggle.MODE_KEY, ModeToggle.LIGHT_MODE); } clearMode() { $('html').removeAttr(ModeToggle.MODE_ATTR); sessionStorage.removeItem(ModeToggle.MODE_KEY); } /* Notify another plugins that the theme mode has changed */ notify() { window.postMessage({ direction: ModeToggle.ID, message: this.modeStatus }, "*"); } } /* ModeToggle */ const toggle = new ModeToggle(); function flipMode() { if (toggle.hasMode) { if (toggle.isSysDarkPrefer) { if (toggle.isLightMode) { toggle.clearMode(); } else { toggle.setLight(); } } else { if (toggle.isDarkMode) { toggle.clearMode(); } else { toggle.setDark(); } } } else { if (toggle.isSysDarkPrefer) { toggle.setLight(); } else { toggle.setDark(); } } toggle.notify(); } /* flipMode() */ </script><body data-spy="scroll" data-target="#toc" data-topbar-visible="true"><div id="sidebar" class="d-flex flex-column align-items-end"><div class="profile-wrapper text-center"><div id="avatar"> <a href="/" alt="avatar" class="mx-auto"> <img src=" /assets/img/ProfilePic.jpg " alt="avatar" onerror="this.style.display='none'"> </a></div><div class="site-title mt-3"> <a href="/">Ferran Cardoso</a></div><div class="site-subtitle font-italic">Personal Profile and <br>Professional Portfolio</div></div><ul class="w-100"><li class="nav-item"> <a href="/" class="nav-link"> <i class="fa-fw fas fa-home ml-xl-3 mr-xl-3 unloaded"></i> <span>HOME</span> </a><li class="nav-item"> <a href="/categories/" class="nav-link"> <i class="fa-fw fas fa-stream ml-xl-3 mr-xl-3 unloaded"></i> <span>CATEGORIES</span> </a><li class="nav-item"> <a href="/archives/" class="nav-link"> <i class="fa-fw fas fa-archive ml-xl-3 mr-xl-3 unloaded"></i> <span>ARCHIVES</span> </a><li class="nav-item"> <a href="/about/" class="nav-link"> <i class="fa-fw fas fa-info-circle ml-xl-3 mr-xl-3 unloaded"></i> <span>ABOUT</span> </a><li class="nav-item"> <a href="/resume/" class="nav-link"> <i class="fa-fw fas fa-tag ml-xl-3 mr-xl-3 unloaded"></i> <span>RESUME</span> </a></ul><div class="sidebar-bottom mt-auto d-flex flex-wrap justify-content-center align-items-center"> <button class="mode-toggle btn" aria-label="Switch Mode"> <i class="fas fa-adjust"></i> </button> <span class="icon-border"></span> <a href="https://github.com/Fercrcode" aria-label="github" target="_blank" rel="noopener"> <i class="fab fa-github"></i> </a> <a href="https://fosstodon.org/@fercr" aria-label="mastodon" target="_blank" rel="noopener"> <i class="fab fa-mastodon"></i> </a> <a href="https://www.linkedin.com/in/fercrprof/" aria-label="linkedin" target="_blank" rel="noopener"> <i class="fab fa-linkedin"></i> </a> <a href=" javascript:location.href = 'mailto:' + ['fercr.post','outlook.com'].join('@')" aria-label="email" > <i class="fas fa-envelope"></i> </a></div></div><div id="topbar-wrapper" class="row justify-content-center"><div id="topbar" class="col-11 d-flex h-100 align-items-center justify-content-between"> <span id="breadcrumb"> <span> <a href="/"> Home </a> </span> <span>Wellcome Single Cell Conference 2022</span> </span> <i id="sidebar-trigger" class="fas fa-bars fa-fw"></i><div id="topbar-title"> Post</div><i id="search-trigger" class="fas fa-search fa-fw"></i> <span id="search-wrapper" class="align-items-center"> <i class="fas fa-search fa-fw"></i> <input class="form-control" id="search-input" type="search" aria-label="search" autocomplete="off" placeholder="Search..."> </span> <span id="search-cancel" >Cancel</span></div></div><div id="main-wrapper"><div id="main"><div class="row"><div id="core-wrapper" class="col-12 col-lg-11 col-xl-8"><div class="post pl-1 pr-1 pl-sm-2 pr-sm-2 pl-md-4 pr-md-4"><h1 data-toc-skip>Wellcome Single Cell Conference 2022</h1><div class="post-meta text-muted"><div> By <em> <a href="https://github.com/fercrcode">Ferran Cardoso Rodriguez</a> </em></div><div class="d-flex"><div> <span> Posted <em class="timeago" data-ts="1654387200" data-toggle="tooltip" data-placement="bottom" data-tooltip-df="llll" > 2022-06-05 </em> </span> <span> Updated <em class="timeago" data-ts="1686345635" data-toggle="tooltip" data-placement="bottom" data-tooltip-df="llll" > 2023-06-09 </em> </span> <span class="readtime" data-toggle="tooltip" data-placement="bottom" title="1294 words"> <em>7 min</em> read</span></div></div></div><div class="post-content"><p><img src="data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 1548 864'%3E%3C/svg%3E" data-src="/assets/img/WellcomeSC22_Pic.jpg" alt="Window shadow" class="shadow" width="1548" height="864" style="max-width: 90%" data-proofer-ignore></p><p>The aim of this post is to improve the accessibility of the talk given at the <strong><a href="https://coursesandconferences.wellcomeconnectingscience.org/event/single-cell-biology-20220606/">Wellcome Single Cell Conference 2022</a></strong> by providing with the notes used for the oral presentation.</p><p>An improved iteration of this talk was given at the <strong>Cancer Institute 2022 conference</strong> as part of the <em>Recent Discoveries</em> series.</p><p>The work presented in this talk eventually developed into that shown in Cardoso Rodriguez &amp; Qin et al. 2023.</p><blockquote class="prompt-info"><div><p>A PDF copy of the slides can be downloaded from <a href="/assets/PDFs/20220606_FerranC.pdf">here</a></p></div></blockquote><h2 id="introduction-110--210"><span class="mr-2">Introduction (1/10 &amp; 2/10)</span><a href="#introduction-110--210" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2></h2><h3 id="intrinsic-and-extrinsic-cues-as-cell-fate-regulators"><span class="mr-2">Intrinsic and Extrinsic Cues as Cell Fate Regulators</span><a href="#intrinsic-and-extrinsic-cues-as-cell-fate-regulators" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h3></h3><p><img src="data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 972 589'%3E%3C/svg%3E" data-src="/assets/img/WellcomeSC22_Cover.png" alt="Desktop View" width="972" height="589" style="max-width: 200px" class="right" data-proofer-ignore> It is understood that both intrinsic and extrinsic cues are crucial in determining cell fate.</p><ul><li>As Intrinsic cues we might have mutations, a cell’s epigenetic state, and transcriptional networks.<li>As for extrinsic cues, we might have mechanobiological signals, metabolites or growth factors.</ul><p>Now, in a cancerous cell we know that intrinsic cues such as oncogenic mutations are key when controlling cell fate, but so are the extrinsic cues coming from the tumour microenvironment.</p><p>Here at the tape lab and in this project in particular are interested in understanding how do oncogenes and the TME regulate cell fate in the context of Colorectal Cancer.</p><h3 id="single-cell-technologies-to-study-crc-organoids"><span class="mr-2">Single-cell Technologies to Study CRC Organoids</span><a href="#single-cell-technologies-to-study-crc-organoids" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h3></h3><p>CRC is a very common cancer, claiming 800 thousand worldwide deaths per year Much like other cancers it presents as a collection of malignant and non-transformed cells, resulting in a complex heterocellular setting where colonic epithelia closely interacts with the surrounding tumour microenvironment, consisting among others of stromal fibroblasts and myeloid macrophages. In fact, TME is key factor in CRC, with increased stromal and myeloid components correlating with worse prognosis.</p><p>In order to study this disease we model it using organoids, self assembling 3D structures that we can grow <em>in vitro</em> in heterocellular settings that can mimic the TME.</p><p>Such a heterogenous system requires of single-cell technologies to be properly characterised. Mass cytometry is easily scalable and ideal to study intra-cellular communications via PTMs. On the other hand, scRNA-seq can be used to describe inter-cellular communication by looking at the expression of ligands and receptors between cells.</p><h2 id="experimental-setup-and-methods-310"><span class="mr-2">Experimental setup and methods (3/10)</span><a href="#experimental-setup-and-methods-310" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2></h2><h3 id="multivariate-scrna-seq-of-crc-tme-organoids"><span class="mr-2">Multivariate scRNA-seq of CRC-TME Organoids</span><a href="#multivariate-scrna-seq-of-crc-tme-organoids" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h3></h3><p>To explore how intrinsic oncogenes and extrinsic stromal cues regulate colonic epithelial fate we performed a multivariate scRNA-seq analysis of organoids.</p><p>In it we had organoids ranging from WT to those progressively harbouring mutational hallmarks of CRC, either in a monoculture or co-cultured with colonic fibroblasts and/or macrophages. Furthermore we also included a WT monoculture grown with exogenous WNT3A, EGF, Nogging and R-spondin1 as a niche factor control.</p><p>Analysed using scRNA-seq we were able to identify the three cell types in the system. We then focused on the epithelial compartment, with the main steps of the analysis shown here; including velocity, differential abundance, and cell-cell communication analysis.</p><h2 id="results-410---910"><span class="mr-2">Results (4/10 - 9/10)</span><a href="#results-410---910" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2></h2><h3 id="scrna-seq-captures-colonic-epithelia-differentiation"><span class="mr-2">scRNA-seq Captures Colonic Epithelia Differentiation</span><a href="#scrna-seq-captures-colonic-epithelia-differentiation" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h3></h3><p>Now with a subset of just of the epithelial cells from the system (around 30k cells across 17 conditions), we were able to capture colonic epithelia differentiation. We identified clusters associated with the main colonic epithelia cell types;<br /> including a stem, secretory and absorptive populations.</p><p>The overall dynamics of the system were also recovered, with SCENT’s CCAT entropy based metric indicating greater pluripotency estimates for the colonic stem cell clusters, as well as RNA velocity stream embeddings going towards the differentiated states. Now, it is important to note that instead of a monolithic colonic stem cells population, we identified a heterogenous compartment with multiple states</p><h3 id="oncogenes-and-fibroblasts-differentially-regulate-csc"><span class="mr-2">Oncogenes and Fibroblasts Differentially Regulate CSC</span><a href="#oncogenes-and-fibroblasts-differentially-regulate-csc" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h3></h3><p>We then investigated when these multiple colonic stem cell states were being regulated using differential abundance analysis. Comparing our control WT monocultures with CRC organoids we saw a significant enrichment of cellular neighbourhoods associated with the proliferative Colonic stem cell population, shown in this PHATE embedding in red. Note also how the more differentiated late enterocyte cells are on the other hand being lost.</p><p>On the other hand, we noticed how the addition of fibroblasts in the co-culture induced a clear enrichment of the revival CSC in the WT organoids.</p><p>These 2 distinct and differentially regulated CSC populations where characterised using Differential Expression.</p><p>As shown in this curated heatmap, proCSC are characterised by being particularly hyperproliferative and expressing <em>Lgr5</em> and <em>Birc5</em>. RevCSC are quite distinct from the other colonic stem cells, being less proliferative and expressing <em>Clusterin</em>; their unique transcriptional signature matching that of the revival stem cell population in the intestine.</p><p>Interestingly, while the CSC and proCSC clusters consist mostly of AK and AKP cells, the revCSC cluster is dominated by WT cells.</p><h3 id="oncogenic-entrapment-of-the-colon-epithelia"><span class="mr-2">Oncogenic Entrapment of the Colon Epithelia</span><a href="#oncogenic-entrapment-of-the-colon-epithelia" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h3></h3><p>That is because it appears that the oncogenic mutations are trapping the colonic epithelia in an oncogenic state.</p><p>First we have a summary of cluster abundance changes across all conditions. It clearly shows how the first 2 colonic stem cell clusters are being upregulated by the oncogenes, and how the revCSC are being induced in WT and ApcKD fibroblast co-cultures.</p><p>However, there’s a clear loss of fibroblast regulation in AK and AKP organoids. Furthermore it seems like WENR is inducing all CSC states in wt organoids.</p><p>We also used RNA velocity lengths to compare differentiation rates across organoids. We compared side-by-side our WT controls, WT+fibroblast cocultures, AK organoids and the WENR enriched condition,</p><p>As you can see here the AK organoids showed the lowest velocity lengths (which was consistent with AKP organoids too).</p><p>To explore the mechanisms behind this oncogenic entrapment we used cell-cell communication analysis of the fibroblast co-cultures across the different genotypes. With these plots we can compare the quantity of incoming and outgoing signals as detected by CellChat, with the arrows showing aggregate interaction strengths from the fibroblasts to the epithelia clusters.</p><p>As we had expected, the fibroblasts appear to be predominantly characterised by outgoing signalling consistently across all genotypes. However, while the WT epithelia shows a high score for incoming signals it quickly dropped following the accumulation of oncogenic mutations.</p><p>These results suggest that oncogenic mutations erode fibroblast to epithelia communication, with the CRC organoids acting as “bad listeners”.</p><h3 id="exogenous-ligands-model-csc-regulation"><span class="mr-2">Exogenous Ligands Model CSC Regulation</span><a href="#exogenous-ligands-model-csc-regulation" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h3></h3><p>(Credit to Dr Xiao Qin)</p><p>Now if you remember we had seen that WENR promoted all CSC states, so to explore how exogenous ligands model CSC regulation we used mass cytometry to analyse organoid monocultures across 60 different conditions with different genotypes and ligands.</p><p>In this condition-based PHATE embedding we can see how:</p><ul><li>We can recapitulate the revCSC cells characterised by <em>Clusterin</em> expression. These cells seem to be associated with exogenous wnt3a and low MAPK and b-cat signalling.<li>We can also see the <em>Birc5</em> positive proCSC , which were characterised by high MAPK and b-cat signalling.</ul><p>Now using a Scaffold map centred around key genotypes we can see how oncogenes and the exogenous ligands navigate a shared regulatory landscape. If for example, we take a WT organoid and add exogenous WNT3A we will be promoting the revCSC. On the other hand, if we take an organoid with just oncogenic Kras (not sufficient on its own to trap the epithelia in a proCSC-like state) and add r-spondin we shift cell fate towards the proCSC.</p><h2 id="summary-1010"><span class="mr-2">Summary (10/10)</span><a href="#summary-1010" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2></h2><p>We have identified 2 distinct and differentially regulated CSC states: Intrinsic oncogenic cues seem to induce a hyper proliferative CSC population Whereas extrinsic stromal cues induce the revCSCs</p><p>Oncogenic mutations are trapping the colonic epithelia in the hyperproliferative state. This seems to be due to the erosion of stromal regulation in CRC organoids. And is also suggested by the lowered differentiation rates in them</p><p>Finally, it appears that both intrinsic and extrinsic cues navigate a shared regulatory landscape Regulated by EGF and WNT signalling, up to the point until APC loss and oncogenic KRAS trap the organoids, turning them refractory towards the external cues</p></div><div class="post-tail-wrapper text-muted"><div class="post-meta mb-3"> <i class="far fa-folder-open fa-fw mr-1"></i> <a href='/categories/conference-event/'>Conference/Event</a>, <a href='/categories/talk/'>Talk</a></div><div class="post-tail-bottom d-flex justify-content-between align-items-center mt-3 pt-5 pb-2"><div class="license-wrapper"> This post is licensed under <a href="https://creativecommons.org/licenses/by/4.0/"> CC BY 4.0 </a> by the author.</div><div class="share-wrapper"> <span class="share-label text-muted mr-1">Share</span> <span class="share-icons"> <a href="https://twitter.com/intent/tweet?text=Wellcome Single Cell Conference 2022 - Ferran Cardoso&amp;url=https://fercrcode.github.io/posts/WellcomeSC22/" data-toggle="tooltip" data-placement="top" title="Twitter" target="_blank" rel="noopener" aria-label="Twitter"> <i class="fa-fw fab fa-twitter"></i> </a> <a href="https://www.facebook.com/sharer/sharer.php?title=Wellcome Single Cell Conference 2022 - Ferran Cardoso&amp;u=https://fercrcode.github.io/posts/WellcomeSC22/" data-toggle="tooltip" data-placement="top" title="Facebook" target="_blank" rel="noopener" aria-label="Facebook"> <i class="fa-fw fab fa-facebook-square"></i> </a> <a href="https://t.me/share/url?url=https://fercrcode.github.io/posts/WellcomeSC22/&amp;text=Wellcome Single Cell Conference 2022 - Ferran Cardoso" data-toggle="tooltip" data-placement="top" title="Telegram" target="_blank" rel="noopener" aria-label="Telegram"> <i class="fa-fw fab fa-telegram"></i> </a> <i id="copy-link" class="fa-fw fas fa-link small" data-toggle="tooltip" data-placement="top" title="Copy link" data-title-succeed="Link copied successfully!"> </i> </span></div></div></div></div></div><div id="panel-wrapper" class="col-xl-3 pl-2 text-muted"><div class="access"><div id="access-lastmod" class="post"><div class="panel-heading">Recently Updated</div><ul class="post-content pl-0 pb-1 ml-1 mt-2"><li><a href="/posts/POPIDD/">Building a Platform for Integrative Discovery and Diagnostics in Cancer</a><li><a href="/posts/MultimodalAP4IDD/">Demonstration of our Multimodal Analytical Platform for the Delivery of Integrated Diagnostics in Oncology</a><li><a href="/posts/EACRpposterIDD/">Presenting the IDD platform at the EACR Cancer Multiomics conference</a><li><a href="/posts/GSPw23/">Graph Signal Processing workshop 2023</a><li><a href="/posts/NanoporeNextflow/">Oxford Nanopore and their EPI2ME nextflow pipelines</a></ul></div></div><script src="https://cdn.jsdelivr.net/gh/afeld/bootstrap-toc@1.0.1/dist/bootstrap-toc.min.js"></script><div id="toc-wrapper" class="pl-0 pr-4 mb-5"><div class="panel-heading pl-3 pt-2 mb-2">Contents</div><nav id="toc" data-toggle="toc"></nav></div></div></div><div class="row"><div class="col-12 col-lg-11 col-xl-8"><div id="tail-wrapper" class="pl-1 pr-1 pl-sm-2 pr-sm-2 pl-md-4 pr-md-4"><div id="related-posts" class="mt-5 mb-2 mb-sm-4"><h3 class="pt-2 mt-1 mb-4 ml-1" data-toc-skip>Further Reading</h3><div class="card-deck mb-4"><div class="card"> <a href="/posts/RecentDiscoveriesCI/"><div class="card-body"> <em class="timeago small" data-ts="1656633600" > 2022-07-01 </em><h3 class="pt-0 mt-1 mb-3" data-toc-skip>Recent Discoveries @ the CI 2022 conference</h3><div class="text-muted small"><p> The aim of this post is to improve the accessibility of the talk given at the Cancer Institute 2022 conference as part of the Recent Discoveries series. This is an improved iteration of the t...</p></div></div></a></div><div class="card"> <a href="/posts/GSPw23/"><div class="card-body"> <em class="timeago small" data-ts="1686787200" > 2023-06-15 </em><h3 class="pt-0 mt-1 mb-3" data-toc-skip>Graph Signal Processing workshop 2023</h3><div class="text-muted small"><p> This is a stub entry discussing my experience at this year’s Graph Signal Processing workshop in Oxford. I attended together with collaborators Aarthi Venkat and Prof. Smita Krishnaswamy, where ...</p></div></div></a></div><div class="card"> <a href="/posts/EACRpposterIDD/"><div class="card-body"> <em class="timeago small" data-ts="1714435200" > 2024-04-30 </em><h3 class="pt-0 mt-1 mb-3" data-toc-skip>Presenting the IDD platform at the EACR Cancer Multiomics conference</h3><div class="text-muted small"><p> This is a stub entry discussing my experience at this year’s EACR Cancer Genomics, Multiomics and Computational Biology conference in Bergamo. Building a Platform for Integrative Discovery and D...</p></div></div></a></div></div></div><div class="post-navigation d-flex justify-content-between"> <a href="/posts/UCLcancerDomain/" class="btn btn-outline-primary" prompt="Older"><p>UCL Cancer Domain Symposium 2022</p></a> <a href="/posts/RecentDiscoveriesCI/" class="btn btn-outline-primary" prompt="Newer"><p>Recent Discoveries @ the CI 2022 conference</p></a></div></div></div></div><footer class="d-flex w-100 justify-content-center"><div class="d-flex justify-content-between align-items-center text-muted"><div class="footer-left"><p class="mb-0"> © 2025 <a href="https://github.com/fercrcode">Ferran Cardoso Rodriguez</a>. <span data-toggle="tooltip" data-placement="top" title="Except where otherwise noted, the blog posts on this site are licensed under the Creative Commons Attribution 4.0 International (CC BY 4.0) License by the author.">Some rights reserved.</span></p></div><div class="footer-right"><p class="mb-0"> Powered by <a href="https://jekyllrb.com" target="_blank" rel="noopener">Jekyll</a> with <a href="https://github.com/cotes2020/jekyll-theme-chirpy" target="_blank" rel="noopener">Chirpy</a> theme.</p></div></div></footer></div><div id="search-result-wrapper" class="d-flex justify-content-center unloaded"><div class="col-12 col-sm-11 post-content"><div id="search-hints"></div><div id="search-results" class="d-flex flex-wrap justify-content-center text-muted mt-3"></div></div></div></div><div id="mask"></div><a id="back-to-top" href="#" aria-label="back-to-top" class="btn btn-lg btn-box-shadow" role="button"> <i class="fas fa-angle-up"></i> </a> <script src="https://cdn.jsdelivr.net/npm/simple-jekyll-search@1.10.0/dest/simple-jekyll-search.min.js"></script> <script> SimpleJekyllSearch({ searchInput: document.getElementById('search-input'), resultsContainer: document.getElementById('search-results'), json: '/assets/js/data/search.json', searchResultTemplate: '<div class="pl-1 pr-1 pl-sm-2 pr-sm-2 pl-lg-4 pr-lg-4 pl-xl-0 pr-xl-0"> <a href="{url}">{title}</a><div class="post-meta d-flex flex-column flex-sm-row text-muted mt-1 mb-1"> {categories} {tags}</div><p>{snippet}</p></div>', noResultsText: '<p class="mt-5">Oops! No result founds.</p>', templateMiddleware: function(prop, value, template) { if (prop === 'categories') { if (value === '') { return `${value}`; } else { return `<div class="mr-sm-4"><i class="far fa-folder fa-fw"></i>${value}</div>`; } } if (prop === 'tags') { if (value === '') { return `${value}`; } else { return `<div><i class="fa fa-tag fa-fw"></i>${value}</div>`; } } } }); </script> <script src="https://cdn.jsdelivr.net/combine/npm/magnific-popup@1/dist/jquery.magnific-popup.min.js,npm/lozad/dist/lozad.min.js,npm/clipboard@2/dist/clipboard.min.js"></script> <script src="https://cdn.jsdelivr.net/combine/npm/dayjs@1/dayjs.min.js,npm/dayjs@1/locale/en.min.js,npm/dayjs@1/plugin/relativeTime.min.js,npm/dayjs@1/plugin/localizedFormat.min.js"></script> <script defer src="/assets/js/dist/post.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/bootstrap@4/dist/js/bootstrap.bundle.min.js"></script> <script defer src="/app.js"></script>
