<!DOCTYPE html><html lang="en" ><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><meta name="generator" content="Jekyll v4.2.2" /><meta property="og:title" content="Demonstration of our Multimodal Analytical Platform for the Delivery of Integrated Diagnostics in Oncology" /><meta property="og:locale" content="en" /><meta name="description" content="Video Demo" /><meta property="og:description" content="Video Demo" /><link rel="canonical" href="https://fercrcode.github.io/posts/MultimodalAP4IDD/" /><meta property="og:url" content="https://fercrcode.github.io/posts/MultimodalAP4IDD/" /><meta property="og:site_name" content="Ferran Cardoso" /><meta property="og:type" content="article" /><meta property="article:published_time" content="2024-12-04T00:00:00+00:00" /><meta name="twitter:card" content="summary" /><meta property="twitter:title" content="Demonstration of our Multimodal Analytical Platform for the Delivery of Integrated Diagnostics in Oncology" /><meta name="twitter:site" content="@FerranC96" /> <script type="application/ld+json"> {"@context":"https://schema.org","@type":"BlogPosting","dateModified":"2024-12-05T10:21:41+00:00","datePublished":"2024-12-04T00:00:00+00:00","description":"Video Demo","headline":"Demonstration of our Multimodal Analytical Platform for the Delivery of Integrated Diagnostics in Oncology","mainEntityOfPage":{"@type":"WebPage","@id":"https://fercrcode.github.io/posts/MultimodalAP4IDD/"},"url":"https://fercrcode.github.io/posts/MultimodalAP4IDD/"}</script><title>Demonstration of our Multimodal Analytical Platform for the Delivery of Integrated Diagnostics in Oncology | Ferran Cardoso</title><link rel="apple-touch-icon" sizes="180x180" href="/assets/img/favicons/apple-touch-icon.png"><link rel="icon" type="image/png" sizes="32x32" href="/assets/img/favicons/favicon-32x32.png"><link rel="icon" type="image/png" sizes="16x16" href="/assets/img/favicons/favicon-16x16.png"><link rel="manifest" href="/assets/img/favicons/site.webmanifest"><link rel="shortcut icon" href="/assets/img/favicons/favicon.ico"><meta name="apple-mobile-web-app-title" content="Ferran Cardoso"><meta name="application-name" content="Ferran Cardoso"><meta name="msapplication-TileColor" content="#da532c"><meta name="msapplication-config" content="/assets/img/favicons/browserconfig.xml"><meta name="theme-color" content="#ffffff"><link rel="preconnect" href="https://fonts.googleapis.com" ><link rel="dns-prefetch" href="https://fonts.googleapis.com" ><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link rel="dns-prefetch" href="https://fonts.gstatic.com" crossorigin><link rel="preconnect" href="https://fonts.googleapis.com" ><link rel="dns-prefetch" href="https://fonts.googleapis.com" ><link rel="preconnect" href="https://cdn.jsdelivr.net" ><link rel="dns-prefetch" href="https://cdn.jsdelivr.net" ><link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Lato&family=Source+Sans+Pro:wght@400;600;700;900&display=swap"><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4/dist/css/bootstrap.min.css"><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.11.2/css/all.min.css"><link rel="stylesheet" href="/assets/css/style.css"><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/afeld/bootstrap-toc@1.0.1/dist/bootstrap-toc.min.css"><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/magnific-popup@1/dist/magnific-popup.min.css"> <script src="https://cdn.jsdelivr.net/npm/jquery@3/dist/jquery.min.js"></script> <script type="text/javascript"> class ModeToggle { static get MODE_KEY() { return "mode"; } static get MODE_ATTR() { return "data-mode"; } static get DARK_MODE() { return "dark"; } static get LIGHT_MODE() { return "light"; } static get ID() { return "mode-toggle"; } constructor() { if (this.hasMode) { if (this.isDarkMode) { if (!this.isSysDarkPrefer) { this.setDark(); } } else { if (this.isSysDarkPrefer) { this.setLight(); } } } let self = this; /* always follow the system prefers */ this.sysDarkPrefers.addEventListener("change", () => { if (self.hasMode) { if (self.isDarkMode) { if (!self.isSysDarkPrefer) { self.setDark(); } } else { if (self.isSysDarkPrefer) { self.setLight(); } } self.clearMode(); } self.notify(); }); } /* constructor() */ get sysDarkPrefers() { return window.matchMedia("(prefers-color-scheme: dark)"); } get isSysDarkPrefer() { return this.sysDarkPrefers.matches; } get isDarkMode() { return this.mode === ModeToggle.DARK_MODE; } get isLightMode() { return this.mode === ModeToggle.LIGHT_MODE; } get hasMode() { return this.mode != null; } get mode() { return sessionStorage.getItem(ModeToggle.MODE_KEY); } /* get the current mode on screen */ get modeStatus() { if (this.isDarkMode || (!this.hasMode && this.isSysDarkPrefer)) { return ModeToggle.DARK_MODE; } else { return ModeToggle.LIGHT_MODE; } } setDark() { $('html').attr(ModeToggle.MODE_ATTR, ModeToggle.DARK_MODE); sessionStorage.setItem(ModeToggle.MODE_KEY, ModeToggle.DARK_MODE); } setLight() { $('html').attr(ModeToggle.MODE_ATTR, ModeToggle.LIGHT_MODE); sessionStorage.setItem(ModeToggle.MODE_KEY, ModeToggle.LIGHT_MODE); } clearMode() { $('html').removeAttr(ModeToggle.MODE_ATTR); sessionStorage.removeItem(ModeToggle.MODE_KEY); } /* Notify another plugins that the theme mode has changed */ notify() { window.postMessage({ direction: ModeToggle.ID, message: this.modeStatus }, "*"); } } /* ModeToggle */ const toggle = new ModeToggle(); function flipMode() { if (toggle.hasMode) { if (toggle.isSysDarkPrefer) { if (toggle.isLightMode) { toggle.clearMode(); } else { toggle.setLight(); } } else { if (toggle.isDarkMode) { toggle.clearMode(); } else { toggle.setDark(); } } } else { if (toggle.isSysDarkPrefer) { toggle.setLight(); } else { toggle.setDark(); } } toggle.notify(); } /* flipMode() */ </script><body data-spy="scroll" data-target="#toc" data-topbar-visible="true"><div id="sidebar" class="d-flex flex-column align-items-end"><div class="profile-wrapper text-center"><div id="avatar"> <a href="/" alt="avatar" class="mx-auto"> <img src=" /assets/img/ProfilePic.jpg " alt="avatar" onerror="this.style.display='none'"> </a></div><div class="site-title mt-3"> <a href="/">Ferran Cardoso</a></div><div class="site-subtitle font-italic">Personal Profile and <br>Professional Portfolio</div></div><ul class="w-100"><li class="nav-item"> <a href="/" class="nav-link"> <i class="fa-fw fas fa-home ml-xl-3 mr-xl-3 unloaded"></i> <span>HOME</span> </a><li class="nav-item"> <a href="/categories/" class="nav-link"> <i class="fa-fw fas fa-stream ml-xl-3 mr-xl-3 unloaded"></i> <span>CATEGORIES</span> </a><li class="nav-item"> <a href="/archives/" class="nav-link"> <i class="fa-fw fas fa-archive ml-xl-3 mr-xl-3 unloaded"></i> <span>ARCHIVES</span> </a><li class="nav-item"> <a href="/about/" class="nav-link"> <i class="fa-fw fas fa-info-circle ml-xl-3 mr-xl-3 unloaded"></i> <span>ABOUT</span> </a><li class="nav-item"> <a href="/resume/" class="nav-link"> <i class="fa-fw fas fa-tag ml-xl-3 mr-xl-3 unloaded"></i> <span>RESUME</span> </a></ul><div class="sidebar-bottom mt-auto d-flex flex-wrap justify-content-center align-items-center"> <button class="mode-toggle btn" aria-label="Switch Mode"> <i class="fas fa-adjust"></i> </button> <span class="icon-border"></span> <a href="https://github.com/Fercrcode" aria-label="github" target="_blank" rel="noopener"> <i class="fab fa-github"></i> </a> <a href="https://fosstodon.org/@fercr" aria-label="mastodon" target="_blank" rel="noopener"> <i class="fab fa-mastodon"></i> </a> <a href="https://www.linkedin.com/in/fercrprof/" aria-label="linkedin" target="_blank" rel="noopener"> <i class="fab fa-linkedin"></i> </a> <a href=" javascript:location.href = 'mailto:' + ['fercr.post','outlook.com'].join('@')" aria-label="email" > <i class="fas fa-envelope"></i> </a></div></div><div id="topbar-wrapper" class="row justify-content-center"><div id="topbar" class="col-11 d-flex h-100 align-items-center justify-content-between"> <span id="breadcrumb"> <span> <a href="/"> Home </a> </span> <span>Demonstration of our Multimodal Analytical Platform for the Delivery of Integrated Diagnostics in Oncology</span> </span> <i id="sidebar-trigger" class="fas fa-bars fa-fw"></i><div id="topbar-title"> Post</div><i id="search-trigger" class="fas fa-search fa-fw"></i> <span id="search-wrapper" class="align-items-center"> <i class="fas fa-search fa-fw"></i> <input class="form-control" id="search-input" type="search" aria-label="search" autocomplete="off" placeholder="Search..."> </span> <span id="search-cancel" >Cancel</span></div></div><div id="main-wrapper"><div id="main"><div class="row"><div id="core-wrapper" class="col-12 col-lg-11 col-xl-8"><div class="post pl-1 pr-1 pl-sm-2 pr-sm-2 pl-md-4 pr-md-4"><h1 data-toc-skip>Demonstration of our Multimodal Analytical Platform for the Delivery of Integrated Diagnostics in Oncology</h1><div class="post-meta text-muted"><div> By <em> <a href="https://github.com/fercrcode">Ferran Cardoso Rodriguez</a> </em></div><div class="d-flex"><div> <span> Posted <em class="timeago" data-ts="1733270400" data-toggle="tooltip" data-placement="bottom" data-tooltip-df="llll" > 2024-12-04 </em> </span> <span> Updated <em class="timeago" data-ts="1733394101" data-toggle="tooltip" data-placement="bottom" data-tooltip-df="llll" > 2024-12-05 </em> </span> <span class="readtime" data-toggle="tooltip" data-placement="bottom" title="2518 words"> <em>13 min</em> read</span></div></div></div><div class="post-content"><h2 id="video-demo"><span class="mr-2">Video Demo</span><a href="#video-demo" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2></h2><div style="text-align: center;"> <iframe width="80%" height="120%" src="https://www.youtube.com/embed/sJU1W3sPwY4?si=adUI5_COabkPxG5A" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen=""></iframe></div><h2 id="structured-transcript"><span class="mr-2">Structured Transcript</span><a href="#structured-transcript" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2></h2><h3 id="poetic-foundry"><span class="mr-2">POETIC-Foundry</span><a href="#poetic-foundry" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h3></h3><p>So this is the landing page from the ICR-Foundry accelerator that was done in late 2023. And as you can see, basically, we have here access to certain key applications that we’ll be showcasing in later. And we also have access to some highlighted resources, including the landing page itself, but also the ontology and the filesystem containing all of the all of the data and also outputs from the analysis here within the Foundry platform. And as you can see down here below, we also have highlighted here an overview of the ontology.</p><h3 id="the-ontology-t-004228"><span class="mr-2">The Ontology [T-00:42:28]</span><a href="#the-ontology-t-004228" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h3></h3><p>So this is essentially a graph based representation of the POETIC clinical trial in which we have basically different layers of information. So these different nodes are coloured according to the type of data. They contain the type of resource that they are within this ontology. And within here we can see the inputs. Essentially, we have like all of the clinical information about the patients, but we also have gene expression information from experiments performed here at the ICR and essentially through a series of transforms and analyses here in the Foundry platform, we were then able to essentially could use a series of outputs. These outputs they can be data access themselves. I mean for subsequent analysis or they can also be applications and widgets. such as this one so here we have, for example, some of the applications that we are showing up here on the top we have the image explorer. We also have the clinical trial data and splitting tool, as well as the sample cohorting analysis tool here and a couple of widgets we will also showcase some of them. in this presentation finally as well, this ontology is useful not just to have an overview of the objects that we have, the kind of data that they contain, but also, for example, to, to showcase ability of foundry to to modulate and in a granular manner to control the permissions that a particular user has to look at it and access the different the different types of data. So for example this is viewing the ontology and colouring of the different nodes as my permission. that I have to view these particular types of data as an example what I can also do is that And for example look at this from the point of view of one of one of my colleagues, which she she has access to the data that was produced in our in our group. This is basically like in biological data regarding histopathological samples, for example. But she will not have access to the clinical data. So if we just view these as as her own, for example, then we’ll see that if we hover over there and some the current nodes, which for example that she’s not able to access patient information, within the ontology, but then she has access to the nodes coloured in blue. So next we would like to show you the three applications that we built using the foundry application.</p><h3 id="clinical-trial-data-splitting-tool-t-031028"><span class="mr-2">Clinical Trial Data Splitting Tool [T-03:10:28]</span><a href="#clinical-trial-data-splitting-tool-t-031028" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h3></h3><p>So the first application is clinical trial data splitting tool where we use this tool to split certain sub cohorts from POETIC into training, set testing and validation sets. And this is for the ease of future collaboration with external collaborators. So here we we have four modules for this application. So the first module we can just input the data set name and requestor, who this will be shared with and however this tool is only accessible by certain management level. So if here me and Ferran to request and define a cohort today, it won’t work. But I can still walk you through each of the module and what we have and what we can do within this module. So the second module is to define the cohort where we can select a certain subgroup of patients based on, you know, different parameters that were included here, such as did they receive peri-operative aromatase inhibitor and what are their baseline breast cancers intrinsic subtypes are they ER-positive / negative or baseline grades, surgery grades. So if we select a certain cohort and then we can then go into the next step, which is sample the cohort, here, we can set the training ratio testing ratio and an evaluation/validation set ratio and then the application will split them based on the ratio, but making sure that the treatment arm and HER2 status is balanced. So it’s like a stratified sampling. And once we have this cohort sampled, we can look at the distribution, then we can see whether we like it or not, if we think everything is good and then going to submit for creation where we can submit the cohort and then whoever the cohort was created for or received notification, and then they can then access only certain type of data. So in this case, the clinical data survival data of the patients will not be visible for those people. And here we can also look have a previously defined cohort and the data of the training testing set splits. However, as I mentioned earlier, we don’t have access to to this application. So we can’t really see anything here and next I would like to show you the second application, The POETIC sample cohorting and analysis</p><h3 id="poetic-sample-cohorting--analysis-t-063438"><span class="mr-2">POETIC Sample Cohorting &amp; Analysis [T-06:34:38]</span><a href="#poetic-sample-cohorting--analysis-t-063438" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h3></h3><p>So this application was built mainly for internal use, for our internal analysts and bioinformaticians And so basically we also first select a sub cohort that we are interested in so we can, you know, look at this cohort and then do some either on the fly analysis or some more sophisticated analysis and similar to the previous application, we can select a subset of patients based on different parameters. For example, received peri-operative aromatase inhibitor or not their HER2 status, maybe we want to look at HER2-positive breast cancer patients. And once we have this, we can then go into analysis. And so in the analyses page it will first have two on the fly analyses including class distribution based on different factors that are selected here and also UMAP of gene expressions for the patients that have gene expression data available. And in this section we can also select previously defined cohorts to review to do more analysis and within the selected cohort we can also further filter out patients that have certain type of data, for example in this case imaging data. And for each patient that is shown here, we can also open a patient 360 view where it takes us to a very comprehensive view of the patient, including the different types of object that links to a patient, some of the basic information from patients and here we can we can see this in member states is where that we have to require additional permission to have a view of the survival data of patients and here we can look at the images that’s available for this patient, including spatial transcriptomic images. H&amp;E slide and Ki67 staining and here we have this direct comparison for clinicopathological variables of the patient at baseline and on-treatment/surgery. And then we can also look at gene expression data where we can search specific genes, for example, here. ESR1 here we can see that the baseline ESR1 as well as at surgery, you can add more genes. For example, MKI67 and we can see that in the on-treatment surgery Ki67 value downregulated after two weeks treatment. And back here we have two more buttons. One is Run Analysis where we… it’s basically running these two analysis and we can change the parameters inside for example the number of k in K-Means clustering. And another button is to perform additional analysis in R, which takes us to Code Workbook application where you can build a pipeline to run analysis on selected cohorts from the cohorting tool this is an example for that. So we have datasets. And R transformation, and then we plotted some plots. So it’s very highly customizable. And yeah for each of the blocks we can have a look at the code that we used and we can also view the plots produced from the code.</p><h3 id="poetic-ml-image-explorer-t-113638"><span class="mr-2">POETIC ML Image Explorer [T-11:36:38]</span><a href="#poetic-ml-image-explorer-t-113638" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h3></h3><p>And next we will show you the final application we built. It’s the POETIC ML machine learning image review So now this application, which could also be accessed through their patient 360 view that Xixuan showed us earlier and can be used to look at the three different types of imaging modalities that we have within these cohort in Foundry. So the first one of those would be their, fluorescent images from spatial transcriptomics datasets that we have, which is again, a small selection of the whole cohort. And then we also have Ki67 IHC whole slide images as well as Hematoxylin and Eosin stain images as well. So the first of all, these modalities, the one that we currently have in here is, is this a spatial transcriptomics technology which is based on looking at the gene expression profile of regions of interest or ROIs. And the idea here is that for all of the patients for which we have these type of data, these data mortality before and after and after surgery, what we can do is we can explore the actual image itself. We can also look at what the different regions of interests were taken. So this is taken with the aid of the pathologist that actually identifies the zones of interest within the tumour location or invasive boundaries and this kind of high level features. And then we can actually explore explore the different information for each of the different ROIs. So, for example, if we’re looking at ROI number number four, and you can actually look in here and you can see that this is like a tumour rich ROI, for example, that has a moderate number of fibroblasts and atypia score of two doesn’t really have a lot of lymphocyte infiltrates in this area. And we also have this button here to get the ROI details, which basically would query a whenever a new image is added into the into the Foundry presentation for POETIC it would query and get access to all of these ROI properties in as well as this modality. Again as I mentioned earlier, we also have staining for Ki67, which is a proliferation or cell division marker. And again you can look at all of the patients that have Ki67 stained slides here. We can also look at the actual or the actual like an overview of the actual image itself. You can look at where the tissue is standing for in these kind of modalities. Is this it would appear as a as a brownish colour. What we do is not just have a button to compute the percentage of positively stained Ki67 cells but you can also compare these directly with the data from hotspot analysis. So this is another kind of of technology that also measures expression of Ki67 but is not based on these actual images themselves. So it’s kind of like an alternative approach. And then you can still see and compare on that patient patient basis. You can see how the two modalities compare. Basically, the scores for these for this proliferation metrics where we can compare across modalities. And as well as looking at these I’m looking at the images and comparing them visually with the hotspot data. We also have a pipeline in the in the backend that computer pixel-wise H score, which essentially looks at them and classifies pixels based on the possible Ki67 intensity levels, and then also get an aggregate score for each of the each of the slides So these can be compared. And to do that actually we also have another of those workbooks that Xixuan showed earlier with within the sample cohorting analysis tool. Finally the last modality hematoxylin and eosin stained slides in here as you can see basically we have some build-in test annotations with the idea that this modality of the imaging basically is very useful to look at overall tissue tissue morphology. And this actually where we can also talk about the functionality within Foundry to flag and tag images or any kind of other data for particular follow up or for certain actions to be taken to the we take I guess So for example, in here we can just talk about, for example, there are some areas here of course, of this light that have mostly fatty tissue. So annotations can be can be created and you can actually select the area with the fatty tissue we want to see that these area we can save this annotation fatty tissue could even change the colour of the of the of the mark for sample we can change it to yellow we can submit this and then we should see basically that not just for for my particular use of viewing this image, but then for any other user that has permission to look at these images, they will then be able to see my own annotations, see who made them, see when they were made. And any other kind of comments that you might want to add them here. And again, for example, if if an image wasn’t for sufficient quality actions could be taken, for example, so for a rescan of the of the image or even like a any other kind of corrective procedure here. And as with the other modalities as well, we have H&amp;E stained images for a considerable number of patients within the POETIC clinical trial. And here, for example, we’ll look at one of our case studies, which is 2210B you can see here as well, this, this beautiful, hematoxylin and eosin stained slide We have two sections of tissue with another a third one containing mostly, mostly fatty tissue. And we’ll use this one basically to to showcase the other kind of more quantitative analysis that could be performed in foundry. So these are powered essentially by Python transformations and Python python packages is we have developed and implemented, which we can run locally however they are running on the cloud in Foundry so these can be run from any kind of device even a thin and light laptop and for example you can use an intensity based approach to detect the tissue in the image. So basically we’ll kind of say what you actually have cellular tissue within these slides and we can also use, for example, a deep learning approach in this case, which is an off the shelf Cellpose model. But again, because the actual backend is written in Python, everything is fully modular and customizable. So when we kind of like in-house, in-house model could be used or an alternative off-the-shelf one, but you also have the ability here.</p></div><div class="post-tail-wrapper text-muted"><div class="post-meta mb-3"> <i class="far fa-folder-open fa-fw mr-1"></i> <a href='/categories/research/'>Research</a>, <a href='/categories/publication/'>Publication</a></div><div class="post-tail-bottom d-flex justify-content-between align-items-center mt-3 pt-5 pb-2"><div class="license-wrapper"> This post is licensed under <a href="https://creativecommons.org/licenses/by/4.0/"> CC BY 4.0 </a> by the author.</div><div class="share-wrapper"> <span class="share-label text-muted mr-1">Share</span> <span class="share-icons"> <a href="https://twitter.com/intent/tweet?text=Demonstration of our Multimodal Analytical Platform for the Delivery of Integrated Diagnostics in Oncology - Ferran Cardoso&amp;url=https://fercrcode.github.io/posts/MultimodalAP4IDD/" data-toggle="tooltip" data-placement="top" title="Twitter" target="_blank" rel="noopener" aria-label="Twitter"> <i class="fa-fw fab fa-twitter"></i> </a> <a href="https://www.facebook.com/sharer/sharer.php?title=Demonstration of our Multimodal Analytical Platform for the Delivery of Integrated Diagnostics in Oncology - Ferran Cardoso&amp;u=https://fercrcode.github.io/posts/MultimodalAP4IDD/" data-toggle="tooltip" data-placement="top" title="Facebook" target="_blank" rel="noopener" aria-label="Facebook"> <i class="fa-fw fab fa-facebook-square"></i> </a> <a href="https://t.me/share/url?url=https://fercrcode.github.io/posts/MultimodalAP4IDD/&amp;text=Demonstration of our Multimodal Analytical Platform for the Delivery of Integrated Diagnostics in Oncology - Ferran Cardoso" data-toggle="tooltip" data-placement="top" title="Telegram" target="_blank" rel="noopener" aria-label="Telegram"> <i class="fa-fw fab fa-telegram"></i> </a> <i id="copy-link" class="fa-fw fas fa-link small" data-toggle="tooltip" data-placement="top" title="Copy link" data-title-succeed="Link copied successfully!"> </i> </span></div></div></div></div></div><div id="panel-wrapper" class="col-xl-3 pl-2 text-muted"><div class="access"><div id="access-lastmod" class="post"><div class="panel-heading">Recently Updated</div><ul class="post-content pl-0 pb-1 ml-1 mt-2"><li><a href="/posts/POPIDD/">Building a Platform for Integrative Discovery and Diagnostics in Cancer</a><li><a href="/posts/MultimodalAP4IDD/">Demonstration of our Multimodal Analytical Platform for the Delivery of Integrated Diagnostics in Oncology</a><li><a href="/posts/EACRpposterIDD/">Presenting the IDD platform at the EACR Cancer Multiomics conference</a><li><a href="/posts/GSPw23/">Graph Signal Processing workshop 2023</a><li><a href="/posts/NanoporeNextflow/">Oxford Nanopore and their EPI2ME nextflow pipelines</a></ul></div></div><script src="https://cdn.jsdelivr.net/gh/afeld/bootstrap-toc@1.0.1/dist/bootstrap-toc.min.js"></script><div id="toc-wrapper" class="pl-0 pr-4 mb-5"><div class="panel-heading pl-3 pt-2 mb-2">Contents</div><nav id="toc" data-toggle="toc"></nav></div></div></div><div class="row"><div class="col-12 col-lg-11 col-xl-8"><div id="tail-wrapper" class="pl-1 pr-1 pl-sm-2 pr-sm-2 pl-md-4 pr-md-4"><div id="related-posts" class="mt-5 mb-2 mb-sm-4"><h3 class="pt-2 mt-1 mb-4 ml-1" data-toc-skip>Further Reading</h3><div class="card-deck mb-4"><div class="card"> <a href="/posts/CSClandscape_Preprint/"><div class="card-body"> <em class="timeago small" data-ts="1676419200" > 2023-02-15 </em><h3 class="pt-0 mt-1 mb-3" data-toc-skip>A Single-cell Perturbation Landscape of Colonic Stem Cell Polarisation</h3><div class="text-muted small"><p> Stub entry for the preprint.</p></div></div></a></div><div class="card"> <a href="/posts/VRland/"><div class="card-body"> <em class="timeago small" data-ts="1678838400" > 2023-03-15 </em><h3 class="pt-0 mt-1 mb-3" data-toc-skip>VR Landscapes</h3><div class="text-muted small"><p> Stub entry for the Valley-Ridge score and single-cell Waddington-like landscapes. Explanation of the aims and tool will go here. This is a link to the code. This is a link to the docs.</p></div></div></a></div><div class="card"> <a href="/posts/pyKrack/"><div class="card-body"> <em class="timeago small" data-ts="1679270400" > 2023-03-20 </em><h3 class="pt-0 mt-1 mb-3" data-toc-skip>Computing Krackhardt hierarchies with pykrack</h3><div class="text-muted small"><p> Stub entry for the pyKrack tool. Explanation of the aims and tool will go here. This is a link to the code. This is a link to the docs.</p></div></div></a></div></div></div><div class="post-navigation d-flex justify-content-between"> <a href="/posts/EACRpposterIDD/" class="btn btn-outline-primary" prompt="Older"><p>Presenting the IDD platform at the EACR Cancer Multiomics conference</p></a> <a href="/posts/POPIDD/" class="btn btn-outline-primary" prompt="Newer"><p>Building a Platform for Integrative Discovery and Diagnostics in Cancer</p></a></div></div></div></div><footer class="d-flex w-100 justify-content-center"><div class="d-flex justify-content-between align-items-center text-muted"><div class="footer-left"><p class="mb-0"> © 2025 <a href="https://github.com/fercrcode">Ferran Cardoso Rodriguez</a>. <span data-toggle="tooltip" data-placement="top" title="Except where otherwise noted, the blog posts on this site are licensed under the Creative Commons Attribution 4.0 International (CC BY 4.0) License by the author.">Some rights reserved.</span></p></div><div class="footer-right"><p class="mb-0"> Powered by <a href="https://jekyllrb.com" target="_blank" rel="noopener">Jekyll</a> with <a href="https://github.com/cotes2020/jekyll-theme-chirpy" target="_blank" rel="noopener">Chirpy</a> theme.</p></div></div></footer></div><div id="search-result-wrapper" class="d-flex justify-content-center unloaded"><div class="col-12 col-sm-11 post-content"><div id="search-hints"></div><div id="search-results" class="d-flex flex-wrap justify-content-center text-muted mt-3"></div></div></div></div><div id="mask"></div><a id="back-to-top" href="#" aria-label="back-to-top" class="btn btn-lg btn-box-shadow" role="button"> <i class="fas fa-angle-up"></i> </a> <script src="https://cdn.jsdelivr.net/npm/simple-jekyll-search@1.10.0/dest/simple-jekyll-search.min.js"></script> <script> SimpleJekyllSearch({ searchInput: document.getElementById('search-input'), resultsContainer: document.getElementById('search-results'), json: '/assets/js/data/search.json', searchResultTemplate: '<div class="pl-1 pr-1 pl-sm-2 pr-sm-2 pl-lg-4 pr-lg-4 pl-xl-0 pr-xl-0"> <a href="{url}">{title}</a><div class="post-meta d-flex flex-column flex-sm-row text-muted mt-1 mb-1"> {categories} {tags}</div><p>{snippet}</p></div>', noResultsText: '<p class="mt-5">Oops! No result founds.</p>', templateMiddleware: function(prop, value, template) { if (prop === 'categories') { if (value === '') { return `${value}`; } else { return `<div class="mr-sm-4"><i class="far fa-folder fa-fw"></i>${value}</div>`; } } if (prop === 'tags') { if (value === '') { return `${value}`; } else { return `<div><i class="fa fa-tag fa-fw"></i>${value}</div>`; } } } }); </script> <script src="https://cdn.jsdelivr.net/combine/npm/magnific-popup@1/dist/jquery.magnific-popup.min.js,npm/lozad/dist/lozad.min.js,npm/clipboard@2/dist/clipboard.min.js"></script> <script src="https://cdn.jsdelivr.net/combine/npm/dayjs@1/dayjs.min.js,npm/dayjs@1/locale/en.min.js,npm/dayjs@1/plugin/relativeTime.min.js,npm/dayjs@1/plugin/localizedFormat.min.js"></script> <script defer src="/assets/js/dist/post.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/bootstrap@4/dist/js/bootstrap.bundle.min.js"></script> <script defer src="/app.js"></script>
