<!DOCTYPE html><html lang="en" ><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><meta name="generator" content="Jekyll v4.2.2" /><meta property="og:title" content="Building a Platform for Integrative Discovery and Diagnostics in Cancer" /><meta property="og:locale" content="en" /><meta name="description" content="Ferran Cardoso`s personal profile, portfolio of academical publications, and other outputs. Uses a minimal, responsive, and powerful Jekyll theme for presenting professional writing." /><meta property="og:description" content="Ferran Cardoso`s personal profile, portfolio of academical publications, and other outputs. Uses a minimal, responsive, and powerful Jekyll theme for presenting professional writing." /><link rel="canonical" href="https://fercrcode.github.io/posts/POPIDD/" /><meta property="og:url" content="https://fercrcode.github.io/posts/POPIDD/" /><meta property="og:site_name" content="Ferran Cardoso" /><meta property="og:type" content="article" /><meta property="article:published_time" content="2025-01-01T00:00:00+00:00" /><meta name="twitter:card" content="summary" /><meta property="twitter:title" content="Building a Platform for Integrative Discovery and Diagnostics in Cancer" /><meta name="twitter:site" content="@FerranC96" /> <script type="application/ld+json"> {"@context":"https://schema.org","@type":"BlogPosting","dateModified":"2025-01-29T11:33:51+00:00","datePublished":"2025-01-01T00:00:00+00:00","description":"Ferran Cardoso`s personal profile, portfolio of academical publications, and other outputs. Uses a minimal, responsive, and powerful Jekyll theme for presenting professional writing.","headline":"Building a Platform for Integrative Discovery and Diagnostics in Cancer","mainEntityOfPage":{"@type":"WebPage","@id":"https://fercrcode.github.io/posts/POPIDD/"},"url":"https://fercrcode.github.io/posts/POPIDD/"}</script><title>Building a Platform for Integrative Discovery and Diagnostics in Cancer | Ferran Cardoso</title><link rel="apple-touch-icon" sizes="180x180" href="/assets/img/favicons/apple-touch-icon.png"><link rel="icon" type="image/png" sizes="32x32" href="/assets/img/favicons/favicon-32x32.png"><link rel="icon" type="image/png" sizes="16x16" href="/assets/img/favicons/favicon-16x16.png"><link rel="manifest" href="/assets/img/favicons/site.webmanifest"><link rel="shortcut icon" href="/assets/img/favicons/favicon.ico"><meta name="apple-mobile-web-app-title" content="Ferran Cardoso"><meta name="application-name" content="Ferran Cardoso"><meta name="msapplication-TileColor" content="#da532c"><meta name="msapplication-config" content="/assets/img/favicons/browserconfig.xml"><meta name="theme-color" content="#ffffff"><link rel="preconnect" href="https://fonts.googleapis.com" ><link rel="dns-prefetch" href="https://fonts.googleapis.com" ><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link rel="dns-prefetch" href="https://fonts.gstatic.com" crossorigin><link rel="preconnect" href="https://fonts.googleapis.com" ><link rel="dns-prefetch" href="https://fonts.googleapis.com" ><link rel="preconnect" href="https://cdn.jsdelivr.net" ><link rel="dns-prefetch" href="https://cdn.jsdelivr.net" ><link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Lato&family=Source+Sans+Pro:wght@400;600;700;900&display=swap"><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4/dist/css/bootstrap.min.css"><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.11.2/css/all.min.css"><link rel="stylesheet" href="/assets/css/style.css"><link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/afeld/bootstrap-toc@1.0.1/dist/bootstrap-toc.min.css"><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/magnific-popup@1/dist/magnific-popup.min.css"> <script src="https://cdn.jsdelivr.net/npm/jquery@3/dist/jquery.min.js"></script> <script type="text/javascript"> class ModeToggle { static get MODE_KEY() { return "mode"; } static get MODE_ATTR() { return "data-mode"; } static get DARK_MODE() { return "dark"; } static get LIGHT_MODE() { return "light"; } static get ID() { return "mode-toggle"; } constructor() { if (this.hasMode) { if (this.isDarkMode) { if (!this.isSysDarkPrefer) { this.setDark(); } } else { if (this.isSysDarkPrefer) { this.setLight(); } } } let self = this; /* always follow the system prefers */ this.sysDarkPrefers.addEventListener("change", () => { if (self.hasMode) { if (self.isDarkMode) { if (!self.isSysDarkPrefer) { self.setDark(); } } else { if (self.isSysDarkPrefer) { self.setLight(); } } self.clearMode(); } self.notify(); }); } /* constructor() */ get sysDarkPrefers() { return window.matchMedia("(prefers-color-scheme: dark)"); } get isSysDarkPrefer() { return this.sysDarkPrefers.matches; } get isDarkMode() { return this.mode === ModeToggle.DARK_MODE; } get isLightMode() { return this.mode === ModeToggle.LIGHT_MODE; } get hasMode() { return this.mode != null; } get mode() { return sessionStorage.getItem(ModeToggle.MODE_KEY); } /* get the current mode on screen */ get modeStatus() { if (this.isDarkMode || (!this.hasMode && this.isSysDarkPrefer)) { return ModeToggle.DARK_MODE; } else { return ModeToggle.LIGHT_MODE; } } setDark() { $('html').attr(ModeToggle.MODE_ATTR, ModeToggle.DARK_MODE); sessionStorage.setItem(ModeToggle.MODE_KEY, ModeToggle.DARK_MODE); } setLight() { $('html').attr(ModeToggle.MODE_ATTR, ModeToggle.LIGHT_MODE); sessionStorage.setItem(ModeToggle.MODE_KEY, ModeToggle.LIGHT_MODE); } clearMode() { $('html').removeAttr(ModeToggle.MODE_ATTR); sessionStorage.removeItem(ModeToggle.MODE_KEY); } /* Notify another plugins that the theme mode has changed */ notify() { window.postMessage({ direction: ModeToggle.ID, message: this.modeStatus }, "*"); } } /* ModeToggle */ const toggle = new ModeToggle(); function flipMode() { if (toggle.hasMode) { if (toggle.isSysDarkPrefer) { if (toggle.isLightMode) { toggle.clearMode(); } else { toggle.setLight(); } } else { if (toggle.isDarkMode) { toggle.clearMode(); } else { toggle.setDark(); } } } else { if (toggle.isSysDarkPrefer) { toggle.setLight(); } else { toggle.setDark(); } } toggle.notify(); } /* flipMode() */ </script><body data-spy="scroll" data-target="#toc" data-topbar-visible="true"><div id="sidebar" class="d-flex flex-column align-items-end"><div class="profile-wrapper text-center"><div id="avatar"> <a href="/" alt="avatar" class="mx-auto"> <img src=" /assets/img/ProfilePic.jpg " alt="avatar" onerror="this.style.display='none'"> </a></div><div class="site-title mt-3"> <a href="/">Ferran Cardoso</a></div><div class="site-subtitle font-italic">Personal Profile and <br>Professional Portfolio</div></div><ul class="w-100"><li class="nav-item"> <a href="/" class="nav-link"> <i class="fa-fw fas fa-home ml-xl-3 mr-xl-3 unloaded"></i> <span>HOME</span> </a><li class="nav-item"> <a href="/categories/" class="nav-link"> <i class="fa-fw fas fa-stream ml-xl-3 mr-xl-3 unloaded"></i> <span>CATEGORIES</span> </a><li class="nav-item"> <a href="/archives/" class="nav-link"> <i class="fa-fw fas fa-archive ml-xl-3 mr-xl-3 unloaded"></i> <span>ARCHIVES</span> </a><li class="nav-item"> <a href="/about/" class="nav-link"> <i class="fa-fw fas fa-info-circle ml-xl-3 mr-xl-3 unloaded"></i> <span>ABOUT</span> </a><li class="nav-item"> <a href="/resume/" class="nav-link"> <i class="fa-fw fas fa-tag ml-xl-3 mr-xl-3 unloaded"></i> <span>RESUME</span> </a></ul><div class="sidebar-bottom mt-auto d-flex flex-wrap justify-content-center align-items-center"> <button class="mode-toggle btn" aria-label="Switch Mode"> <i class="fas fa-adjust"></i> </button> <span class="icon-border"></span> <a href="https://github.com/Fercrcode" aria-label="github" target="_blank" rel="noopener"> <i class="fab fa-github"></i> </a> <a href="https://fosstodon.org/@fercr" aria-label="mastodon" target="_blank" rel="noopener"> <i class="fab fa-mastodon"></i> </a> <a href="https://www.linkedin.com/in/fercrprof/" aria-label="linkedin" target="_blank" rel="noopener"> <i class="fab fa-linkedin"></i> </a> <a href=" javascript:location.href = 'mailto:' + ['fercr.post','outlook.com'].join('@')" aria-label="email" > <i class="fas fa-envelope"></i> </a></div></div><div id="topbar-wrapper" class="row justify-content-center"><div id="topbar" class="col-11 d-flex h-100 align-items-center justify-content-between"> <span id="breadcrumb"> <span> <a href="/"> Home </a> </span> <span>Building a Platform for Integrative Discovery and Diagnostics in Cancer</span> </span> <i id="sidebar-trigger" class="fas fa-bars fa-fw"></i><div id="topbar-title"> Post</div><i id="search-trigger" class="fas fa-search fa-fw"></i> <span id="search-wrapper" class="align-items-center"> <i class="fas fa-search fa-fw"></i> <input class="form-control" id="search-input" type="search" aria-label="search" autocomplete="off" placeholder="Search..."> </span> <span id="search-cancel" >Cancel</span></div></div><div id="main-wrapper"><div id="main"><div class="row"><div id="core-wrapper" class="col-12 col-lg-11 col-xl-8"><div class="post pl-1 pr-1 pl-sm-2 pr-sm-2 pl-md-4 pr-md-4"><h1 data-toc-skip>Building a Platform for Integrative Discovery and Diagnostics in Cancer</h1><div class="post-meta text-muted"><div> By <em> <a href="https://github.com/fercrcode">Ferran Cardoso Rodriguez</a> </em></div><div class="d-flex"><div> <span> Posted <em class="timeago" data-ts="1735689600" data-toggle="tooltip" data-placement="bottom" data-tooltip-df="llll" > 2025-01-01 </em> </span> <span> Updated <em class="timeago" data-ts="1738150431" data-toggle="tooltip" data-placement="bottom" data-tooltip-df="llll" > 2025-01-29 </em> </span> <span class="readtime" data-toggle="tooltip" data-placement="bottom" title="1131 words"> <em>6 min</em> read</span></div></div></div><div class="post-content"><p><img src="data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 1548 864'%3E%3C/svg%3E" data-src="/assets/img/EHAA25.png" alt="Window shadow" class="shadow" width="1548" height="864" style="max-width: 90%" data-proofer-ignore></p><p>The aim of this post is to improve the accessibility of the talks given at the <strong><a href="https://www.eventbrite.co.uk/e/2nd-networking-event-empowering-healthcare-with-automated-analysis-tickets-1207054696089?aff=oddtdtcreator">2nd Networking Event: Empowering Healthcare with Automated Analysis</a></strong> by providing with the notes used for the oral presentation.</p><blockquote class="prompt-info"><div><p>A PDF copy of the slides can be downloaded from <a href="/assets/PDFs/20250129_LdnMet.pdf">here</a></p></div></blockquote><h2 id="introduction"><span class="mr-2">Introduction</span><a href="#introduction" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2></h2><h3 id="presentation"><span class="mr-2">Presentation</span><a href="#presentation" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h3></h3><p>Hello everyone, after Carlos’ wonderful and thought-provoking talk I will try and present within mine and the collected efforts of the Integrated Pathology Unit towards building a platform for Integrative Discovery and Diagnosis in Cancer. My presenter notes and a copy of the slides can be found on the site by scanning this QR code here.</p><h3 id="background"><span class="mr-2">Background</span><a href="#background" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h3></h3><p>So, to begin with let me talk a bit about myself. I did a bachelor’s in Biotechnology at the University of Barcelona, during the last year of which I actually came to London to work on my thesis. There I realised two things: first, that London was a wonderful place and that I wanted to stay there, secondly, that I found much more enjoyment analysing the data I was generating in the wet lab. It was then that I decided to pivot towards fully computational roles, doing a master’s in bioinformatics and theoretical systems biology at Imperial and a PhD in computational biology at UCL’s Cancer Institute. During my PhD I worked at Chris Tape’s lab towards understanding colorectal cancer biology and chemotherapy resistance using single-cell omic technologies and organoids as model. I worked on developing and publishing tools for mass cytometry data analysis, published a piece on the phenoscape of colonic stem cell regulation together with Xiao Qin (a former postdoc in Tape Lab that now leads her own lab in Oxford), and worked on exploring the use of knowledge graphs for capturing and interrogating cellular communications.</p><h3 id="the-integrated-pathology-unit"><span class="mr-2">The Integrated Pathology Unit</span><a href="#the-integrated-pathology-unit" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h3></h3><p>Now as I introduced earlier, I am part of the Integrated Pathology Unit, together with my colleague Priya and having had the pleasure to host Carlos for his sabbatical from City during 2024. The IPU is actually a very unique setting, nestled in between the research-focused Institute of Cancer Research, and the clinically-focused Royal Marsden Hospital.</p><p>At the IPU we work not just with digital pathology images from different modalities, from H&amp;E to IHC and multiplex immunofluorescence, but are also incorporating higher dimensional technologies like Celldive and spatial transcriptomics. All in all, there is a veritable wealth of data passing through the IPU, especially considering that we work with most types of solid tumour cancer, but as many of you here know, massive amounts of data don’t necessarily equate to knowledge. In our particular setting, this challenge is compounded by the need to work with this many data modalities, so are setting out to develop our own analysis platform, POPIDD.</p><h2 id="popidd"><span class="mr-2">POPIDD</span><a href="#popidd" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2></h2><p>POPIDD, or Pathomics Platform for Integrative Discovery and Diagnostics represents our effort towards a solution able to work with not only the imaging modalities I just described, but also the associated clinical data so information can be extracted (including feature extraction and analysis) and integrated (using common data structures), ultimately leading to an increased understanding of clinical and biological processes. This is a growing framework centred around a modular backend (being developed almost exclusively in Python), so that we can consistently reuse in-house tools and also easily incorporate off-the-shelf models. Additionally, the IPU is a wet-lab heavy group, so being very conscious of the need for accessibility in empowering non-computational colleagues, we are also extending on the open source high-dimensional data viewer Napari to build an interface for POPIDD.</p><h3 id="backend"><span class="mr-2">Backend</span><a href="#backend" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h3></h3><p>POPIDDs main pipeline remains unreleased beyond internal use for now, but elements like napari plugins and python packages (that can work either in a standalone way or as part of POPIDD) have already been released. See here popid-io, a napari plugin for reading digital pathology images and annotations as GeoJSONs and parquet files, or pyhscore, a simple package to compute pixel-wise H-scores. Again, as I mentioned earlier, we are focused on building accessible interfaces for these tools, so on the following slides I will show you some use cases of the custom napari interface accessible from POPIDD’s main pipeline.</p><h3 id="frontend"><span class="mr-2">Frontend</span><a href="#frontend" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h3></h3><p>This is the interface, with custom widgets on the right. See for example on the upper right the batch loading popid-io widget, loading the shown mIF image, its channel metadata on the left, and adjusting the viewers scale bar. POPIDD’s interface incorporates a series of tabs for the different main components of the tool, we will begin here by talking about the tissue and cell segmentation tab, which uses both in-house and third party models. I also wanted to show here how we integrate standalone packages, in this case our own pyhscore package, that can be used to quickly quantify IHC images. Finally, there is also a module for aligning TMAs, so their cores can be mapped to the appropriate sample or patient ID before downstream analysis. The widget here uses the Python implementation of this pipeline, but those of you that came to CBIAS might remember how we also talked about a matlab version (perhaps a bit more performant than this one …) developed by Carlos.</p><h3 id="application-on-epi700-cohort"><span class="mr-2">Application on Epi700 cohort</span><a href="#application-on-epi700-cohort" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h3></h3><p>I now wanted to take the last slide of my talk to show a real-life use case of POPIDD in the context of analysing a clinical colorectal cancer cohort. Some of you might find this familiar as it is very similar to the work I presented some months ago at CBIAS. Based on robust signal processing methods, TMAAP identifies the core layout on TMA slides and maps them to their relevant patient and sample IDs. Mapping alignment scores inform the confidence of the ID assignations, which can be visually inspected through a customised interface of the multi-dimensional image viewer napari. We validate TMAAP’s performance by applying it to the Epi700 colorectal cancer cohort, consisting of over 30 thousand individual immunohistochemistry (IHC) and H&amp;E core images. TMAAP can accurately map these images to tumour and stromal samples for each patient. Following downstream IHC quantification, we then recapitulate known clinical and molecular features across all patients; including expected biological interactions between IHC targets and greater degrees of immune involvement in microsatellite instable and BRAF mutant tumours.</p><h2 id="closing-remarks"><span class="mr-2">Closing remarks</span><a href="#closing-remarks" class="anchor text-muted"><i class="fas fa-hashtag"></i></a></h2></h2><p>Hopefully by now I have shown the idea that, not only is it crucial for us to develop, share, and use great tools advancing our research, but also that we have to be especially mindful of accessibility. Therefore, using graphical interfaces bypasses in my experience one of the main barriers to non-computational colleagues; the dreaded command line interface. I also wanted to take this time to highlight the IPU’s software repository, where myself, Priya and Carlos have been working on these tools and releasing some of them.</p><p>Thank you for your time and acknowledge all great people!</p></div><div class="post-tail-wrapper text-muted"><div class="post-meta mb-3"> <i class="far fa-folder-open fa-fw mr-1"></i> <a href='/categories/research/'>Research</a>, <a href='/categories/conference-event/'>Conference/Event</a></div><div class="post-tail-bottom d-flex justify-content-between align-items-center mt-3 pt-5 pb-2"><div class="license-wrapper"> This post is licensed under <a href="https://creativecommons.org/licenses/by/4.0/"> CC BY 4.0 </a> by the author.</div><div class="share-wrapper"> <span class="share-label text-muted mr-1">Share</span> <span class="share-icons"> <a href="https://twitter.com/intent/tweet?text=Building a Platform for Integrative Discovery and Diagnostics in Cancer - Ferran Cardoso&amp;url=https://fercrcode.github.io/posts/POPIDD/" data-toggle="tooltip" data-placement="top" title="Twitter" target="_blank" rel="noopener" aria-label="Twitter"> <i class="fa-fw fab fa-twitter"></i> </a> <a href="https://www.facebook.com/sharer/sharer.php?title=Building a Platform for Integrative Discovery and Diagnostics in Cancer - Ferran Cardoso&amp;u=https://fercrcode.github.io/posts/POPIDD/" data-toggle="tooltip" data-placement="top" title="Facebook" target="_blank" rel="noopener" aria-label="Facebook"> <i class="fa-fw fab fa-facebook-square"></i> </a> <a href="https://t.me/share/url?url=https://fercrcode.github.io/posts/POPIDD/&amp;text=Building a Platform for Integrative Discovery and Diagnostics in Cancer - Ferran Cardoso" data-toggle="tooltip" data-placement="top" title="Telegram" target="_blank" rel="noopener" aria-label="Telegram"> <i class="fa-fw fab fa-telegram"></i> </a> <i id="copy-link" class="fa-fw fas fa-link small" data-toggle="tooltip" data-placement="top" title="Copy link" data-title-succeed="Link copied successfully!"> </i> </span></div></div></div></div></div><div id="panel-wrapper" class="col-xl-3 pl-2 text-muted"><div class="access"><div id="access-lastmod" class="post"><div class="panel-heading">Recently Updated</div><ul class="post-content pl-0 pb-1 ml-1 mt-2"><li><a href="/posts/POPIDD/">Building a Platform for Integrative Discovery and Diagnostics in Cancer</a><li><a href="/posts/MultimodalAP4IDD/">Demonstration of our Multimodal Analytical Platform for the Delivery of Integrated Diagnostics in Oncology</a><li><a href="/posts/EACRpposterIDD/">Presenting the IDD platform at the EACR Cancer Multiomics conference</a><li><a href="/posts/GSPw23/">Graph Signal Processing workshop 2023</a><li><a href="/posts/NanoporeNextflow/">Oxford Nanopore and their EPI2ME nextflow pipelines</a></ul></div></div><script src="https://cdn.jsdelivr.net/gh/afeld/bootstrap-toc@1.0.1/dist/bootstrap-toc.min.js"></script><div id="toc-wrapper" class="pl-0 pr-4 mb-5"><div class="panel-heading pl-3 pt-2 mb-2">Contents</div><nav id="toc" data-toggle="toc"></nav></div></div></div><div class="row"><div class="col-12 col-lg-11 col-xl-8"><div id="tail-wrapper" class="pl-1 pr-1 pl-sm-2 pr-sm-2 pl-md-4 pr-md-4"><div id="related-posts" class="mt-5 mb-2 mb-sm-4"><h3 class="pt-2 mt-1 mb-4 ml-1" data-toc-skip>Further Reading</h3><div class="card-deck mb-4"><div class="card"> <a href="/posts/VRland/"><div class="card-body"> <em class="timeago small" data-ts="1678838400" > 2023-03-15 </em><h3 class="pt-0 mt-1 mb-3" data-toc-skip>VR Landscapes</h3><div class="text-muted small"><p> Stub entry for the Valley-Ridge score and single-cell Waddington-like landscapes. Explanation of the aims and tool will go here. This is a link to the code. This is a link to the docs.</p></div></div></a></div><div class="card"> <a href="/posts/pyKrack/"><div class="card-body"> <em class="timeago small" data-ts="1679270400" > 2023-03-20 </em><h3 class="pt-0 mt-1 mb-3" data-toc-skip>Computing Krackhardt hierarchies with pykrack</h3><div class="text-muted small"><p> Stub entry for the pyKrack tool. Explanation of the aims and tool will go here. This is a link to the code. This is a link to the docs.</p></div></div></a></div><div class="card"> <a href="/posts/NanoporeNextflow/"><div class="card-body"> <em class="timeago small" data-ts="1682899200" > 2023-05-01 </em><h3 class="pt-0 mt-1 mb-3" data-toc-skip>Oxford Nanopore and their EPI2ME nextflow pipelines</h3><div class="text-muted small"><p> This is a stub entry for some of my work helping a collaborator sequence and run some Oxford Nanopore data. The entry will include work troubleshooting a MinION Mk1C device, and exploring the next...</p></div></div></a></div></div></div><div class="post-navigation d-flex justify-content-between"> <a href="/posts/MultimodalAP4IDD/" class="btn btn-outline-primary" prompt="Older"><p>Demonstration of our Multimodal Analytical Platform for the Delivery of Integrated Diagnostics in Oncology</p></a> <span class="btn btn-outline-primary disabled" prompt="Newer"><p>-</p></span></div></div></div></div><footer class="d-flex w-100 justify-content-center"><div class="d-flex justify-content-between align-items-center text-muted"><div class="footer-left"><p class="mb-0"> © 2025 <a href="https://github.com/fercrcode">Ferran Cardoso Rodriguez</a>. <span data-toggle="tooltip" data-placement="top" title="Except where otherwise noted, the blog posts on this site are licensed under the Creative Commons Attribution 4.0 International (CC BY 4.0) License by the author.">Some rights reserved.</span></p></div><div class="footer-right"><p class="mb-0"> Powered by <a href="https://jekyllrb.com" target="_blank" rel="noopener">Jekyll</a> with <a href="https://github.com/cotes2020/jekyll-theme-chirpy" target="_blank" rel="noopener">Chirpy</a> theme.</p></div></div></footer></div><div id="search-result-wrapper" class="d-flex justify-content-center unloaded"><div class="col-12 col-sm-11 post-content"><div id="search-hints"></div><div id="search-results" class="d-flex flex-wrap justify-content-center text-muted mt-3"></div></div></div></div><div id="mask"></div><a id="back-to-top" href="#" aria-label="back-to-top" class="btn btn-lg btn-box-shadow" role="button"> <i class="fas fa-angle-up"></i> </a> <script src="https://cdn.jsdelivr.net/npm/simple-jekyll-search@1.10.0/dest/simple-jekyll-search.min.js"></script> <script> SimpleJekyllSearch({ searchInput: document.getElementById('search-input'), resultsContainer: document.getElementById('search-results'), json: '/assets/js/data/search.json', searchResultTemplate: '<div class="pl-1 pr-1 pl-sm-2 pr-sm-2 pl-lg-4 pr-lg-4 pl-xl-0 pr-xl-0"> <a href="{url}">{title}</a><div class="post-meta d-flex flex-column flex-sm-row text-muted mt-1 mb-1"> {categories} {tags}</div><p>{snippet}</p></div>', noResultsText: '<p class="mt-5">Oops! No result founds.</p>', templateMiddleware: function(prop, value, template) { if (prop === 'categories') { if (value === '') { return `${value}`; } else { return `<div class="mr-sm-4"><i class="far fa-folder fa-fw"></i>${value}</div>`; } } if (prop === 'tags') { if (value === '') { return `${value}`; } else { return `<div><i class="fa fa-tag fa-fw"></i>${value}</div>`; } } } }); </script> <script src="https://cdn.jsdelivr.net/combine/npm/magnific-popup@1/dist/jquery.magnific-popup.min.js,npm/lozad/dist/lozad.min.js,npm/clipboard@2/dist/clipboard.min.js"></script> <script src="https://cdn.jsdelivr.net/combine/npm/dayjs@1/dayjs.min.js,npm/dayjs@1/locale/en.min.js,npm/dayjs@1/plugin/relativeTime.min.js,npm/dayjs@1/plugin/localizedFormat.min.js"></script> <script defer src="/assets/js/dist/post.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/bootstrap@4/dist/js/bootstrap.bundle.min.js"></script> <script defer src="/app.js"></script>
